bundles/action/org.openhab.action.satel/.project

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.openhab.action.satel</name>
+	<comment>This is the ${binding-name} binding of the open Home Automation Bus (openHAB)</comment>
```

Comment:
This is generated comment, most of action bundles has tha same comment. Also (as far as I know) this is not shown anywhere, IMO not important, but will correct it.